src/Components/Assets/AssetServiceEditModal.tsx

```diff
@@ -170,18 +170,30 @@ export const AssetServiceEditModal = (props: {
               data-testid="asset-last-serviced-on-input"
             >
               <FieldLabel>Serviced On</FieldLabel>
```

Comment:
You shouldn't be using `FieldLabel` when using `FormField` components. Use the `label` prop of the `FormField` component instead. This is to keep designs consistent.